### DISCUSSION TITLE
Non-managed devices with custom clients

### DISCUSSION THREAD

**Discussion by qinamical-tec** [rustdesk/rustdesk-server-pro#354] [Q&A]
Sometimes we need to deploy a custom client for a quick support session.
We do not want that device to get added to the managed devices list, as it has been used just once with a custom client that was not even installed.
There should be a configuration for custom clients to be unmanaged so that they do not get added to the address book.

**Comment by rustdesk** [comment]
Apologize, we won't do this. we have given our solution in https://github.com/rustdesk/rustdesk-server-pro/discussions/182
TeamViewer / AnyDesk can do this, because you are using their cloud solution, free users and pro users share the same public servers. This can not be done on self-hosting solution. You can see this if our cloud solution is out in the future.
